lms/lmsweb/tools/registration.py

```diff
@@ -129,6 +167,22 @@ def _build_user_text(user: UserToCreate) -> str:
         return msg
 
 
+def generate_confirmation_token(email: str) -> str:
+    return SERIALIZER.dumps(email, salt='email-confirmation')
+
+
+def send_confirmation_mail(email: str, fullname: str) -> None:
+    token = generate_confirmation_token(email)
+    msg = Message(
+        'Confirmation Email - Learn Python',
```

Comment:
`_()` on both this part and the message. 
+ Maybe RTL support?

The `Learning Python` part should be taken from configuration `site_name` or something